contracts/evoting/mod_test.go

```diff
@@ -1100,6 +1102,119 @@ func TestCommand_DecryptBallots(t *testing.T) {
 	require.True(t, election.DecryptedBallots[1].Equal(expectedBallot2))
 }
 
+func TestCommand_CombineSharesCanLogToDela(t *testing.T) {
+	// setup a fake logger for the test purpose
+	logBuffer := new(bytes.Buffer)
+
+	logWriter := zerolog.ConsoleWriter{
+		Out:        logBuffer,
+		TimeFormat: time.RFC3339,
+	}
+
+	dela.Logger = zerolog.New(logWriter).Level(zerolog.InfoLevel)
```

Comment:
You don't need a  `ConsoleWriter`. You can directly use `zerolog.New(logBuffer)`. This will yield JSON, which makes it easier to compare and parse.

Reply:
When a test changes a global variable, it is always best to set it back:

```go
oldLog := dela.Logger
dela.Logger = ...
defer func() {
  dela.Logger = oldLog
}()
```

Because otherwise the next test won't have the expected logger.

Reply:
👍